### PR TITLE
Bug 1120821

### PR DESCRIPTION
**Pull request by pemmasanikrishna**

### CODE REVIEW

**Comment by groovecoder** [comment]
Thanks for the contribution. Can you update [the corresponding test](https://github.com/pemmasanikrishna/kuma/blob/5f3cef4a4f6fd131516c8f85a1609691ba1c0439/kuma/users/tests/test_models.py#L20-L63) with some of the new values that should pass validation now?

**Comment by pemmasanikrishna** [comment]
Hi, @groovecoder  can you tell why the travis failed.

**Comment by groovecoder** [comment]
We have an issue where the tests on Travis sometimes fail to reach the `media/uploads/` directory. It's a transient error so we just restart the job. :unamused: I'll restart it.

File: kuma/users/tests/test_models.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def test_websites(self):
             twitter='http://twitter.com/lmorchard',
             github='http://github.com/lmorchard',
             stackoverflow='http://stackoverflow.com/users/lmorchard',
-            linkedin='https://www.linkedin.com/in/testuser',
+            linkedin='https://in.linkedin.com/in/testuser',
```
Comment (groovecoder) [inline kuma/users/tests/test_models.py]: Can you make the test try multiple linkedin profile field values? I.e., `https://www.linkedin.com/in/testuser`, `https://in.linkedin.com/in/testuser`, `https://www.linkedin.com/pub/testuser`

**Comment by groovecoder** [comment]
Please amend the commit message to be more descriptive. E.g., "Bug 1120821 - change linkedin url regex for user model"

**Comment by pemmasanikrishna** [comment]
Same problem , I think it should be restarted again. :(

**Comment by groovecoder** [comment]
This one is a real error:
```
======================================================================

ERROR: Failure: SyntaxError (keyword argument repeated (test_models.py, line 36))

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/build/mozilla/kuma/vendor/packages/nose/nose/loader.py", line 413, in loadTestsFromName

    addr.filename, addr.module)

  File "/home/travis/build/mozilla/kuma/vendor/packages/nose/nose/importer.py", line 47, in importFromPath

    return self.importFromDir(dir_path, fqname)

  File "/home/travis/build/mozilla/kuma/vendor/packages/nose/nose/importer.py", line 94, in importFromDir

    mod = load_module(part_fqname, fh, filename, desc)

  File "/home/travis/build/mozilla/kuma/kuma/users/tests/test_models.py", line 36

    linkedin='https://www.linkedin.com/in/testuser',

SyntaxError: keyword argument repeated
```

**Comment by groovecoder** [comment]
To run this test locally, run `./manage.py test kuma.users.tests.test_models:TestUserProfile.test_websites`

**Comment by pemmasanikrishna** [comment]
Is that caused because I have included all the test cases at once in test file?

**Comment by groovecoder** [comment]
Yes. You can't repeat keyword arguments in python function calls. You should either add multiple calls to save & verify the profile in that test, or add another test specifically for the linkedin values.

**Comment by pemmasanikrishna** [comment]
@groovecoder  How can do that, need some help

**Comment by groovecoder** [comment]
@pemmasanikrishna - how much python & django experience do you have? It's a pretty basic refactor of the test to call the [model's save method](https://docs.djangoproject.com/en/1.7/ref/models/instances/#saving-objects) more than once, and to assert the saved value [equals the expected value](https://nose.readthedocs.org/en/latest/testing_tools.html#tools-for-testing).

**Comment by pemmasanikrishna** [comment]
I have experience in python but not in Django.

**Comment by groovecoder** [comment]
Okay, that model save method link above should help you learn about how it works.

**Comment by pemmasanikrishna** [comment]
i have looked the link but should I add the save method immediately in the test case? and where should I assert the value?

**Comment by groovecoder** [comment]
@pemmasanikrishna - check out http://chimera.labs.oreilly.com/books/1234000000754/ch03.html, it should help you understand the order of test code in python.

**Comment by groovecoder** [comment]
@pemmasanikrishna - are you still planning to finish this or can I add a commit onto it for the test code part?

**Comment by pemmasanikrishna** [comment]
@groovecoder  The regex check is working locally fine for me, but I don't have idea of creating a test case for it. can you do it for me.

**Comment by groovecoder** [comment]
Superseded by https://github.com/mozilla/kuma/pull/3317 with test.

**Pull request closed without merging**